.github/workflows/linux.yml

```diff
@@ -29,29 +29,37 @@ jobs:
         include:
           - label: PostgreSQL 10
             postgresql-version: "10"
+            runs-os: ubuntu-20.04
           - label: PostgreSQL 11
             postgresql-version: "11"
+            runs-os: ubuntu-20.04
           - label: PostgreSQL 12
             postgresql-version: "12"
+            runs-os: ubuntu-20.04
           - label: PostgreSQL 13
             postgresql-version: "13"
+            runs-os: ubuntu-20.04
           - label: PostgreSQL 14
             postgresql-version: "14"
+            runs-os: ubuntu-20.04
           - label: PostgreSQL 14 with Groonga master
             groonga-master: "yes"
             postgresql-version: "14"
+            runs-os: ubuntu-20.04
           - label: PostgreSQL 15 snapshot
             postgresql-unreleased: "yes"
             postgresql-version: "15"
+            runs-os: ubuntu-22.10
           - label: PostgreSQL 15 snapshot with Groonga master
             groonga-master: "yes"
             postgresql-unreleased: "yes"
             postgresql-version: "15"
+            runs-os: ubuntu-22.10
```

Comment:
22.04?
It seems that Ubuntu 22.10 isn't released yet.

Reply:
No.
The version of postgresql-common on 22.04 is 238.
https://packages.ubuntu.com/jammy/postgresql-common

Ubuntu 22.10 isn't released yet but, Docker Hub has already Ubuntu 22.10.
https://hub.docker.com/_/ubuntu

The version of postgresql-common on 22.10 is `241ubuntu1`.
https://packages.ubuntu.com/kinetic/postgresql-common

Reply:
`runs-on` isn't for Docker image.
It's for GitHub Actions' runner: https://github.com/actions/virtual-environments

Reply:
Oh...
I didn't know it...

Thank you for your comment.